decompiler/extractor/main.cpp

```diff
@@ -357,7 +371,11 @@ int main(int argc, char** argv) {
     setup_global_decompiler_stuff(std::nullopt);
   }
 
-  std::filesystem::path path_to_iso_files = file_util::get_jak_project_dir() / "iso_data" / "jak1";
+  std::filesystem::path path_to_iso_files = file_util::get_jak_project_dir() / "iso_data" / "_temp";
+  if (flag_extract && data_dir_path != path_to_iso_files) {
+    // in case input is also output, don't just wipe everything (weird)
+    std::filesystem::remove_all(path_to_iso_files);
```

Comment:
I think this and the `create_directories` call below should just be moved into the `flag_extract` check below